### PR TITLE
Remove non_empty_domain_var() Fragment Info PyBind11 Function

### PR DESCRIPTION
**Pull request by nguyenv**
Remove non_empty_domain_var() Fragment Info PyBind11 Function

    * This has now been consolidated into get_non_empty_domain().

### CODE REVIEW

File: tiledb/tests/test_libtiledb.py [hunk 1]
```diff
@@ -1927,15 +1907,7 @@ def test_varlen_wrong_domain(self):
...
-        A = np.array(
-            [
-                b"aa",
-                b"bbb",
-                b"cccc",
-                np.uint64([1, 3, 4]),
-            ],
-            dtype=object,
-        )
+        A = np.array([b"aa", b"bbb", b"cccc", np.uint64([1, 3, 4])], dtype=object)
```
Comment (ihnorton) [inline tiledb/tests/test_libtiledb.py:1910]: (This is what I was talking about w/ black :/ -- this file has been formatted before, no reason for it to change now)

File: tiledb/fragment.cc [hunk 2]
```diff
@@ -109,6 +109,13 @@ class PyFragmentInfo {
 
   template <typename T>
   py::tuple get_non_empty_domain(py::object schema, uint32_t fid, T did) const {
+    py::bool_ isvar = get_dim_isvar(schema.attr("domain"), did);
```
Comment (ihnorton) [inline tiledb/fragment.cc:112]: Does this need to be templated? Seems like it is only called for `uint32_t`

**Review by ihnorton — APPROVED** [review]
Minor question, but LGTM

File: tiledb/tests/test_libtiledb.py (hunk 1 above)
Comment (nguyenv) [reply tiledb/tests/test_libtiledb.py:1910]: Sigh yeah and there seems to be some incongruence between the version of Black being used for GitHub Actions and what I have installed locally. I set it up to always use the latest stable version, so I don't know which one isn't up to date. I also didn't realize how frequently they make changes to this format.

File: tiledb/fragment.cc (hunk 2 above)
Comment (nguyenv) [reply tiledb/fragment.cc:112]: Yes it should be able to take in both the dimension's index the dimension's name.

File: tiledb/tests/test_libtiledb.py (hunk 1 above)
Comment (Shelnutt2) [reply tiledb/tests/test_libtiledb.py:1910]: Does black have a way to introduce a format file so its consistent in the repo across versions that change the default?

Comment (ihnorton) [reply tiledb/tests/test_libtiledb.py:1910]: Last time I looked it seems they always do it The Correct Way so no configuration ... except that way changes :)

File: tiledb/fragment.cc (hunk 2 above)
Comment (ihnorton) [reply tiledb/fragment.cc:112]: I see, thanks 👍

File: tiledb/tests/test_libtiledb.py (hunk 1 above)
Comment (eddelbuettel) [reply tiledb/tests/test_libtiledb.py:1910]: Maybe you can tie the call to Black down in a Docker container to freeze it.  Seems like a heavy hammerl though.

**Pull request merged**